Sources/PackageModel/Manifest.swift

```diff
@@ -69,3 +69,14 @@ extension Manifest {
         return try JSON(string: PackageDescription.jsonString(package: package)).toString(prettyPrint: true)
     }
 }
+
+extension Manifest: Hashable {
```

Comment:
Is the name really the right thing to be checking here?  I would expect manifests to use identity equality; just because two manifests have the same name doesn't mean that they are the same one.